src/map/utils/petutils.cpp

```diff
@@ -619,6 +619,190 @@ namespace petutils
         }
     }
 
+    void LoadTrustStats(CTrustEntity* PTrust)
+    {
+        // Cargo cult of PC calculations.
+
+        float raceStat = 0;         // конечное число HP для уровня на основе расы.
+        float jobStat = 0;          // конечное число HP для уровня на основе первичной профессии.
+        float sJobStat = 0;         // коенчное число HP для уровня на основе вторичной профессии.
+        int32 bonusStat = 0;            // бонусное число HP которое добавляется при соблюдении некоторых условий.
+
+        int32 baseValueColumn = 0;  // номер колонки с базовым количеством HP
+        int32 scaleTo60Column = 1;  // номер колонки с модификатором до 60 уровня
+        int32 scaleOver30Column = 2;    // номер колонки с модификатором после 30 уровня
+        int32 scaleOver60Column = 3;    // номер колонки с модификатором после 60 уровня
+        int32 scaleOver75Column = 4;    // номер колонки с модификатором после 75 уровня
+        int32 scaleOver60 = 2;          // номер колонки с модификатором для расчета MP после 60 уровня
+        int32 scaleOver75 = 3;          // номер колонки с модификатором для расчета Статов после 75-го уровня
+
+        uint8 grade;
+
+        uint8 mlvl = PTrust->GetMLevel();
+        uint8 slvl = PTrust->GetSLevel();
+        JOBTYPE mjob = PTrust->GetMJob();
+        JOBTYPE sjob = PTrust->GetSJob();
+
+        uint8 race = 0;                 //Human
+
+        switch (PTrust->look.race)
```

Comment:
I was going to use a mobfamily because there are trusts who don't fit player races
liek this https://github.com/TeoTwawki/darkstar/commit/6c4f2fd8e230b03a8d117b3491c5a2baa0948dfe#diff-f70a2d82e222d87eff0e2b0598b2d4b0R18